bin/regenerate-offline

```diff
@@ -6,14 +6,21 @@ GOVUK_FRONTEND_PREFIX=https://github.com/alphagov/govuk-frontend/releases/downlo
 GOVUK_FRONTEND_ARCHIVE_NAME=release-v${RELEASE}.zip
 GOVUK_FRONTEND_URL="${GOVUK_FRONTEND_PREFIX}${GOVUK_FRONTEND_ARCHIVE_NAME}"
 
-wget ${GOVUK_FRONTEND_URL}
+# Test file exists and is a regular file
```

Comment:
AFAIK this tests the file _doesn't_ exist

Reply:
Updated comment to reflect correct logic of what is tested